trie/proof.go

```diff
@@ -63,7 +63,7 @@ func (t *Trie) Prove(key []byte, fromLevel uint, proofDb ethdb.KeyValueWriter) e
 			var err error
 			tn, err = t.resolveHash(n, prefix)
 			if err != nil {
-				log.Error(fmt.Sprintf("Unhandled trie error: %v", err))
+				log.Error("Unhandled trie error in Trie.Prove", "err", err)
```

Comment:
This is not unhandled, it's passed to the caller

Reply:
should we nuke out this log?

Reply:
Let's leave it for now, to see if it's hit